src/Mbstring/Resources/unidata/caseFolding.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+return [
```

Comment:
Shall we add some credits for tchwork/utf8's work?

Reply:
Definitely! Not sure which form should it take however. Maybe:

```php
// This map comes from tchwork/utf8 repository (https://github.com/tchwork/utf8).
```

?

Reply:
no need for credits about this, the map is just derived unicode data